### ISSUE TITLE
Indexing not possible

### ISSUE THREAD

**Issue by firmenpunkt** [tpwd/ke_search#260]
In the versions 5.7.3 and above the CType changed. Since then I cannot select the Storage Folder (Records Storage Page), where the Indexer configuration and the filters are stored. In earlier versions, I could select this folder in the Search Box configuration. I guess this is the reason the indexer does not find anything. So this description is not correct anymore: https://docs.typo3.org/p/tpwd/ke_search/6.1/en-us/QuickStart/Index.html (the arrow pointing to the Records Storage Page; it has been replaced with Starting Point)
![ke-searchbox-conf](https://github.com/user-attachments/assets/125bdfe7-a09c-4c91-b23e-70d2e9c11fa4)
![ke-indexing](https://github.com/user-attachments/assets/77ca88b5-4202-4e13-ad0b-87ae6b0870a7)

**Comment by christianbltr** [comment]
The field "Record Storage Page" has been renamed by the core to "Startingpoint" (see screenshot below). But it's still the same field (internally `pages`) and the functionality has not changed.

![Auswahl_017](https://github.com/user-attachments/assets/e0b5c203-fff3-4df2-97d4-5bf892b71422)

Also you are mixing up two things here: Setting the startingpoint in the searchbox plugin does affect the listed indexer configurations which are shown in the backend module in "Start indexing". And also the CType of the searchbox plugin does not affect the listed indexer configurations. There must be some other reason why the "No indexer configurations have been found" message is shown. Maybe the indexer configurations are disabled (set to "hidden")?

**Comment by firmenpunkt** [comment]
OMG I found it finally!
the Overridestartingpont in the setup.typoscript pointed to the wrong page, I commented it out, as I don't need it. Thanks a lot for the help.
Sorry for my questions, the new setup after the updates was a bit confusing :)
I would like to ask you if you have time to adjust the quick start part of the manual:
https://docs.typo3.org/p/tpwd/ke_search/6.1/en-us/QuickStart/Index.html
